operator-framework-core/src/test/java/io/javaoperatorsdk/operator/processing/event/EventSourceManagerTest.java

```diff
@@ -94,16 +95,16 @@ void retrievingEventSourceForClassShouldWork() {
   }
 
   @Test
-  void shouldNotBePossibleToAddEventSourcesForSameTypeAndQualifier() {
+  void shouldNotBePossibleToAddEventSourcesForSameTypeAndName() {
     EventSourceManager manager = initManager();
 
     CachingEventSource eventSource = mock(CachingEventSource.class);
-    when(eventSource.getResourceClass()).thenReturn(String.class);
+    when(eventSource.getResourceClass()).thenReturn(InformerEventSource.class);
```

Comment:
That's not correct. `getResourceClass` is supposed to return the class handled by the event source. So if you specify that the resource class is `InformerEventSource`, this means that your event source deals with `InformerEventSources` which doesn't really make sense. Granted, `String` doesn't make a lot more sense either but it's a little bit more realistic and doesn't risk clashing with the `EventSource` class itself (which should be reserved for cases where there is no associated type with an `EventSource`).

Reply:
Ohh, yes, sorry my fault, will change it!

Reply:
fixed